internal/server/data/data.go

```diff
@@ -511,6 +513,24 @@ func GlobalCount[T models.Modelable](tx GormTxn, selectors ...SelectorFunc) (int
 	return count, nil
 }
 
+// QuickGlobalCount gives the count of all records, not scoped by org, and not filterable, but very fast.
+func QuickGlobalCount[T models.Modelable](tx GormTxn) (int64, error) {
+	db := tx.GormDB()
+
+	s, err := schema.Parse((*T)(nil), &sync.Map{}, schema.NamingStrategy{})
+	if err != nil {
+		return 0, fmt.Errorf("trying to get table name: %w", err)
+	}
+	tableName := s.Table
+
+	var count int64
+	if err := db.Raw("SELECT reltuples AS count FROM pg_class WHERE relname = ?", tableName).Scan(&count).Error; err != nil {
```

Comment:
This returns the wrong count as it includes the deleted groups

Reply:
well, deleted anything, really. I'm not sure there's an easy way around this given we're doing soft deletes.

Reply:
Right, any resource that has soft delete. It's not worth making this change if it counts the deleted resources since the count will have no meaning

Reply:
I mean it's kind of nice for growth analytics. you don't really want deleted records to reduce your count of how much the product is getting used. :D 
These counts with conditions all have to go long term, as they'll eventually start to take minutes and 10s of minutes to run. There's some more complicated alternatives like moving deleted records to separate tables, or maintaining the count yourself in redis.

Reply:
Definitely, I think global counts become not that interesting pretty quickly. We will hopefully start to be a lot more interested in things like "number of users and groups per org", and "number of active orgs". I'd expect us to get that from our analytics platform, or exporting the data to an analytics database which can support this type of query.